lib/resources/package.rb

```diff
@@ -45,36 +47,32 @@ def initialize(package_name = nil, opts = {}) # rubocop:disable Metrics/AbcSize
       elsif ['hpux'].include?(os[:family])
         @pkgman = HpuxPkg.new(inspec)
       else
-        return skip_resource 'The `package` resource is not supported on your OS yet.'
+        raise Inspec::Exceptions::ResourceSkipped, 'The `package` resource is not supported on your OS yet.'
```

Comment:
Thanks for this - in the same vein as my prior comment, since we're adopting exceptions as the new normal, moving toward that is a good thing.